wandb/sdk/artifacts/artifact.py

```diff
@@ -501,6 +475,44 @@ def metadata(self, metadata: dict) -> None:
         """
         self._metadata = self._normalize_metadata(metadata)
 
+    @property
+    def ttl(self) -> Union[datetime.timedelta, ArtifactTTL, None]:
+        """Artifact TTL(time to live).
+
+        Once the artifact's created_at is older than the ttl, the artifact is marked as deleted.
+        To turn off previously set artifact TTL, reset the TTL to wandb.ArtifactTTL.DEFAULT
+        EX: Artifact.created_at = 1/1/2000, ttl=10 days.
+            This artifact will be marked for deletion on 1/11/2000.
```

Comment:
Nit: this is our public facing documentation. Let's be a little more scrupulous.